docs/apidocs.fsx

```diff
@@ -169,6 +169,63 @@ type GenericClass2<'T>() =
 /// and <see cref="M:TheNamespace.GenericClass2`1.GenericMethod``1(`0,``0)" />
 let referringFunction2 () = "result"
 
+(**
+### Classic XMl Doc Comments: Excluding APIs from the docs
+
+If you want to exclude modules or functions from the API docs you can use the `<exclude/>` tag.
+It needs to be set on a separate triple-slashed line, and can either appear on its own or as part
+of an existing `<summary>` (for example, you may wish to hide existing documentation while it's in progress).
+The `<exclude/>` tag can be the first or last line in these cases.
+
+Some examples:
+*)
+
+/// </exclude>
```

Comment:
Should this not be `<exclude />`? `</exclude>` without opening tag doesn't strike me as valid XML.

Reply:
Yes, a typo. Sorry :-/